apps/server/src/modules/board/board.module.ts

```diff
@@ -51,6 +57,11 @@ import { ColumnBoardCopyService } from './service/column-board-copy.service';
 		BoardDoCopyService,
 		ColumnBoardCopyService,
 		SchoolSpecificFileCopyServiceFactory,
+		// Media Board
```

Comment:
I would prefer to have a separate media-board.module.ts, but the current architecture does not really allow for it, so this helps to separate the module in a later refactoring.

Reply:
...then a seperate array would be better instead of a comment imo, because there could be something after the comment in the future which doesnt belongs to media board